src/sites/twitch-twilight/modules/chat/viewer_card.jsx

```diff
@@ -52,6 +62,7 @@ export default class ViewerCards extends Module {
 	onEnable() {
 		this.chat.context.on('changed:chat.viewer-cards.highlight-chat', this.refreshStyle, this);
 		this.chat.context.on('changed:chat.viewer-cards.color', this.refreshStyle, this);
+		this.css_tweaks.toggle('hide-gift-subscribe-button', this.settings.get('chat.viewer-cards.hide-gift-subscribe-button'));
```

Comment:
This should be
```javascript
this.chat.context.getChanges('chat.viewer-cards.hide-gift-subscribe-button', val =>
    this.css_tweaks.toggleHide('hide-gift-subscribe-button', val)
);
```

And also, rather than adding a CSS blob for this hide rule, the hide rule should be added to the `CLASSES` object in css_tweaks, which generates `display: none` for rules.